tools/splat_ext/mapfs.yaml

```diff
@@ -0,0 +1,1040 @@
+- [dro_01_shape, geom_shape]
```

Comment:
What's the reason for adding the type (e.g. `geom_shape`) to this list? The type can always(?) be inferred from the name suffix and the engine assumes particular suffixes

Reply:
Because I wanted to make it like the same functionality from splat and I didn't know the engine could take care of identifying types by names so I added it for this reason. I'll just remove the type metadata completely...

Reply:
Fair reasoning, but yeah, no need :+1: